Assets/Scripts/NPCs/States/FleeState.cs

```diff
@@ -6,11 +6,16 @@ public class FleeState : CultistBaseState
 {
     private Transform _enemyTransform;
 
-    private float _fleeDuration = 1.5f; // Duration the cultist should stay in FleeState
+    private float _fleeDuration = 3f; // Duration the cultist should stay in FleeState
```

Comment:
[SerializeField] to edit in inspector

Reply:
Made an SO variable in cultist data SO for this instead of serialize field.